.circleci/config.yml

```diff
@@ -7,11 +7,22 @@ version: 2.1
 orbs:
   maven: circleci/maven@1.0.3
 
+machine: true
+
 jobs:
   run_checkstyle:
     <<: *defaults
     working_directory: /tmp/repos/cmo-metadb
+    environment:
+      - TESTCONTAINERS_RYUK_DISABLED: true
     steps:
+      - setup_remote_docker:
+          version: 20.10.2
```

Comment:
you don't need a remote Docker if you are using `machine: true`. In fact, that's most probably why it doesn't work :)
```suggestion
```

Reply:
thanks! yeah I've tried several variations with and without machine + with/without installing docker manually

Reply:
i *do* think that the solution is getting the right docker installation going inside of the container, will update when I've played around with it a bit more

Reply:
and D O N E 🙌